core/templates/pages/exploration-player-page/learner-experience/continue-button.component.html

```diff
@@ -0,0 +1,26 @@
+<button mat-raised-button class="oppia-learner-confirm-button protractor-test-continue-to-next-card-button"
+        [oppiaFocusOn] = "'::' + focusLabel"
+        (click)="clickContinueButton.emit()"
+        aria-label="Continue">
+  <div *ngIf="!isLearnAgainButton">
+    <span class="protractor-test-next-card-button" [innerHTML]="'I18N_PLAYER_CONTINUE_BUTTON' | translate"></span>
+    <i class="fas fa-arrow-right icon">
+    </i>
+  </div>
+  <div *ngIf="isLearnAgainButton">
+    <span class="protractor-test-next-card-button" [innerHTML]="'I18N_PLAYER_LEARN_AGAIN_BUTTON' | translate"></span>
+    <i class="fas fa-redo icon">
+    </i>
+  </div>
+</button>
+
+<style>
+  continue-button .oppia-learner-confirm-button {
+    margin-top: 12px;
+  }
+
+  continue-button .oppia-learner-confirm-button .icon {
```

Comment:
@srijanreddy98, FYI, Using the directive name for the selector is working fine for @EricZLou.